<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GitCommitDialog</class>
 <widget class="QWidget" name="GitCommitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Git</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="logGroup">
     <property name="title">
      <string>Commit Message</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="logEdit">
        <property name="toolTip">
         <string>Enter the log message.</string>
        </property>
        <property name="whatsThis">
         <string>&lt;b&gt;Log Message&lt;/b&gt;
&lt;p&gt;Enter the log message for the commit action.&lt;/p&gt;</string>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Recent commit messages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="recentComboBox">
        <property name="toolTip">
         <string>Select a recent commit message to use</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="stagedCheckBox">
     <property name="toolTip">
      <string>Select to commit only staged changes</string>
     </property>
     <property name="text">
      <string>Commit staged changes only</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="amendCheckBox">
     <property name="toolTip">
      <string>Select to amend the last commit (leave message empty to keep it)</string>
     </property>
     <property name="text">
      <string>Amend the last commit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="resetAuthorCheckBox">
     <property name="toolTip">
      <string>Select to reset the author information</string>
     </property>
     <property name="text">
      <string>Reset Author Info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>logEdit</tabstop>
  <tabstop>recentComboBox</tabstop>
  <tabstop>stagedCheckBox</tabstop>
  <tabstop>amendCheckBox</tabstop>
  <tabstop>resetAuthorCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
